pkg/openservicebroker/cmd/server/start.go

```diff
@@ -13,10 +13,13 @@ import (
 	"github.com/openshift/origin/pkg/openservicebroker/server"
 )
 
-const defaultEtcdPathPrefix = "/registry/templateservicebroker.openshift.io"
-
 type TemplateServiceBrokerServerOptions struct {
-	RecommendedOptions *genericoptions.RecommendedOptions
+	// we don't have any storage, so we shouldn't use the recommended options
+	SecureServing  *genericoptions.SecureServingOptions
+	Authentication *genericoptions.DelegatingAuthenticationOptions
+	Authorization  *genericoptions.DelegatingAuthorizationOptions
+	Audit          *genericoptions.AuditOptions
```

Comment:
I'm guessing this is making our audit harder, of course. Although the advanced audit is designed to work with multiple api server I'm not 100% sure if that was tested ;) Btw. do we care about upstream audit options here. We haven't decided yet which parts of the advanced audit we will enable and this is literally taking what upstream has.